omnibus/config/software/oc_bifrost.rb

```diff
@@ -25,7 +25,7 @@
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
-  profile_name = fips_mode? ? "fips" : "default"
+  profile_name = "default"
```

Comment:
I assumed it would do that but wanted to introduce as few changes as possible. I'll go ahead and remove this though since it does `default` by... default 😁